evaluators/rank/NdcgEvaluator/README.md

```diff
@@ -0,0 +1,4 @@
+# NDCGEvaluator
+
+`NDCGEvaluator` measures the performance of a retrieval system based on the graded relevance of the predicted scores and desired scores. It ranges from `0.0` to `1.0`, with `1.0` representing the ideal ranking of the retrieved result.
```

Comment:
Add at the end.

`Make sure to adapt `id_tag` from RankEvaluateDriver to a field in document tags containing the relevance scores`